tests/lib/release-ops.js

```diff
@@ -479,4 +483,62 @@ describe("ReleaseOps", () => {
         });
     });
 
+    describe("writeChangelog", () => {
+
+        const cwd = process.cwd();
+        let sandbox = null;
+
+        beforeEach(() => {
+            sandbox = sinon.sandbox.create();
+            const tmpDir = fs.mkdtempSync(path.join(os.tmpdir(), "writeChangelog-"));
```

Comment:
It might be good to remove this directory in `afterEach` so that running the tests doesn't leave temporary directories behind.

Reply:
Done in fc24c3372d2563a043df47de38dd6cf4b2e92b38.